--- mozillavpn-2.11.0~rc8/src/connectionbenchmark/benchmarktaskdownload.cpp	2022-10-31 16:00:04.486905207 +0000
+++ mozillavpn-2.11.0~rc8/src/connectionbenchmark/benchmarktaskdownload.cpp	2022-10-31 16:00:06.858877007 +0000
@@ -47,11 +47,11 @@
     m_dnsLookup.setNameserver(QHostAddress(MULLVAD_DEFAULT_DNS));
 #endif
 
+#if defined(MVPN_ANDROID) || defined(MVPN_WASM)
 #if QT_VERSION >= 0x060500
 #  error Check if QT added support for QDnsLookup::lookup() on Android
 #endif
 
-#if defined(MVPN_ANDROID) || defined(MVPN_WASM)
     NetworkRequest* request =
         NetworkRequest::createForGetUrl(this, m_fileUrl.toString());
     connectNetworkRequest(request);
--- mozillavpn-2.11.0~rc8/src/connectionbenchmark/benchmarktasktransfer.cpp     2022-11-15 16:44:34.537527089 +0000
+++ mozillavpn-2.11.0~rc8/src/connectionbenchmark/benchmarktasktransfer.cpp     2022-11-15 16:44:37.845540228 +0000
@@ -53,7 +53,7 @@
     m_dnsLookup.lookup();
 #endif
 
-#if QT_VERSION >= 0x060400
+#if defined(MVPN_ANDROID) && QT_VERSION >= 0x060400
 #  error Check if QT added support for QDnsLookup::lookup() on Android
 #endif

